assets/scss/tools/_image.scss

```diff
@@ -0,0 +1,21 @@
+// http://sassmeister.com/gist/1b4f2da5527830088e4d
+@function get-padding($size) {
```

Comment:
Can we add some doc blocks for these helpers which we are adding. Also this would be a good candidate for documentation.